controllers/chaosimpl/timechaos/impl.go

```diff
@@ -69,6 +69,8 @@ func (impl *Impl) Apply(ctx context.Context, index int, records []*v1alpha1.Reco
 		Sec:         sec,
 		Nsec:        nsec,
 		ClkIdsMask:  mask,
+		Uid:         string(obj.GetUID()) + string(decodedContainer.Pod.GetUID()),
+		PodId:       string(decodedContainer.Pod.GetUID()),
```

Comment:
What will happen if I start two `TimeChaos` to inject different containers in the same pod? Is it expected for them to have same `PodId`?

Reply:
Sorry , I will fix it. When I start the work , it is not a problem because Timechaos have no container selector.